tests/features/custom_page/add_custom_page.feature

```diff
@@ -41,3 +46,27 @@ Feature: "Add custom page" visibility options.
     # Check that the collection content such as the 'Join collection block' is
     # available in context of the custom page.
     Then I should see the link "Leave this collection"
+
+    # I should not be able to add a custom page to a different collection
+    When I go to the homepage of the "Code Camp" collection
+    Then I should not see the link "Add custom page"
```

Comment:
Just a question, the comment says "I should not be able to add a custom page to a different collection".
The button of "Add custom page" does not have this meaning (we have seen that there were cases that buttons appeared when they shouldn't or vice versa).

But anyway, my question is, do we need this tested in behat since it's already covered in Og tests? I mean the permission overlapping between groups.